locales/ar-MA.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
If they are empty, can we remove them?

Reply:
Sure, I'll wait nuxt pr to fix wrong merge... 

I'll remove also unused `es` country variants files and update contributing guide

Reply:
done: PR merged on nuxt i18n module, not yet released, we can merge this, we can change `en-US.json`and update contributing guide later